src/components/ReturnsCalculator.jsx

```diff
@@ -0,0 +1,244 @@
+		} else if (value > 70) {
+			setSliderBG("red.400");
+		} else {
+			setSliderBG("yellow.300");
+		}
+	};
+
+	return (
+		<React.Fragment>
+			<Helmet>
+				<title>YieldScan - Returns Calculator</title>
+				<meta name='description' content='Validator key stats' />
+			</Helmet>
+			<LogEvent eventType='Returns calculator view' />
+			<Route exact path='/returns-calculator'>
+				<Heading mt={16} mb={12} 
+					ml={"calc(15% - 2rem)" // 992px upwards
+					}>
+					Calculate your returns
+				</Heading>
+				<Flex alignItems='center' flexWrap='wrap' justify="center">
+					<Box minWidth='288px' mr={8}>
+						<Box
+							color='gray.500'
+							fontWeight='semibold'
+							letterSpacing='wide'
+							fontSize='md'
+							mb={8}
+						>
+							<Flex flexWrap='wrap'>
+								<Flex direction='column' mr={8} mb={4}>
+									<Text mb={2}>I want to spend</Text>
+									<Input
+										placeholder='Enter your budget'
+										variant='filled'
+										color={textColor[props.colorMode]}
+										type='number'
+										min='0'
+										step='0.000000000001'
+										max='999999999999999'
+										value={stakeInput}
+										textAlign='center'
+										rounded='40px'
+										minWidth='299px'
+										onChange={e => {
+											setStakeInput(parseFloat(e.target.value));
+										}}
+									/>
+								</Flex>
+								<Flex direction='column'>
+									<Text mb={2}>Currency</Text>
+									<InputGroup>
+										<InputRightAddon
+											align='center'
+											px={8}
+											children='KSM'
+											backgroundColor='#4A5567'
+											color='white'
+											roundedRight='40px'
+											roundedLeft='40px'
+										/>
+									</InputGroup>
+								</Flex>
+							</Flex>
+						</Box>
+
+						<Box>
+							<Text
+								color='gray.500'
+								fontWeight='semibold'
+								letterSpacing='wide'
+								fontSize='md'
+								mt={8}
+								mb={4}
+							>
+								With Risk Level
+							</Text>
+							<Slider
+								defaultValue={50}
+								value={riskLevel}
+								onChange={value => {
+									setRiskLevel(value);
+									onRiskChange(value);
+								}}
+							>
+								<SliderTrack />
+								<SliderFilledTrack bg={sliderBG} />
+								<SliderThumb />
+							</Slider>
+							<Flex
+								justifyContent='space-between'
+								width='100%'
+								color='gray.500'
+								letterSpacing='wide'
+								fontSize='sm'
+								mb={8}
+							>
+								<Text>Low</Text>
+								<Text>Medium</Text>
+								<Text>High</Text>
+							</Flex>
+						</Box>
+						<CustomButton
+							onClick={calculate}
+						>
+							Calculate
+						</CustomButton>
+					</Box>
+					<Box
+						m={4}
+						w={[
+							"100%", // base
+							"100%", // 480px upwards
+							"calc(50% - 2rem)", // 768px upwards
+							"calc(40% - 2rem)" // 992px upwards
+						]}
+					>
+						<ExpectedReturns
+							returns={expectedReturns}
+							currency={'KSM'}
```

Comment:
Use the currency variable defined by Abhinav

Reply:
Changed.